### DISCUSSION TITLE
how do you get the galaxy brain badge

### DISCUSSION THREAD

**Discussion by DdosTester25** [community/community#18965] [Other Feature Feedback, Questions, & Ideas] [answered]
already got the pull shark one, but confused about the one badge I genuinely want

**Comment by justin-2028** [comment, marked as answer]
Check this link for more information:
https://github.com/Schweinepriester/github-profile-achievements

Remember, your 2 answers have to be accepted.

**Reply by DdosTester25** [reply]
oh wait then my comments were all for nothing
